deeptime/markov/msm/markov_state_model.py

```diff
@@ -1087,77 +1100,49 @@ def score(self, dtrajs, score_method='VAMP2', score_k=10):
         ----------
         dtrajs : list of arrays
             test data (discrete trajectories).
-        score_method : str
-            Overwrite scoring method if desired. If `None`, the estimators scoring
-            method will be used. See __init__ for documentation.
-        score_k : int or None
+        dim : int or None
             Overwrite scoring rank if desired. If `None`, the estimators scoring
             rank will be used. See __init__ for documentation.
-        score_method : str, optional, default='VAMP2'
-            Overwrite scoring method to be used if desired. If `None`, the estimators scoring
-            method will be used.
-            Available scores are based on the variational approach for Markov processes
-            :cite:`msm-noe2013variational` :cite:`msm-wu2020variational`:
+        r : float or str, optional, default=2
+            Overwrite scoring method to be used if desired. Can be any float greater or equal 1 or 'E' for VAMP-r score
+            or VAMP-E score, respectively.
 
-            * 'VAMP1': Sum of singular values of the symmetrized transition matrix :cite:`msm-wu2020variational`.
+            Special cases :cite:`msm-noe2013variational` :cite:`msm-wu2020variational`:
+
+            * 'VAMP1': Sum of singular values of the symmetrized transition matrix.
```

Comment:
It is here basically the same as above. it sounds like r could be 'VAMP1' or 'VAMP2'